pages/documentation/developer/api_guide/single-line-diagram_svg-writing.md

```diff
@@ -0,0 +1,214 @@
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-cgmes-conversion</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>com.powsybl</groupId>
+        <artifactId>powsybl-triple-store-impl-rdf4j</artifactId>
+        <version>${powsybl.core.version}</version>
+    </dependency>
+    <dependency>
+        <groupId>org.slf4j</groupId>
+        <artifactId>slf4j-simple</artifactId>
+        <version>${slf4j.version}</version>
+    </dependency>
+</dependencies>
+
+<properties>
+    <powsybl.sld.version>1.7.2</powsybl.sld.version>
+    <powsybl.core.version>3.7.1</powsybl.core.version>
+    <slf4j.version>1.7.22</slf4j.version>
+</properties>
+```
+
+Here are some details about these dependencies:
+- `powsybl-single-line-diagram-core` is the core module of single-line-diagram,
+- `powsybl-iidm-impl` is for the Network model,
+- `powsybl-iidm-test` is for loading the test Network,
+- `powsybl-config-test` is for loading the test configuration (see [configuration API guide](configuration.md)),
+- `powsybl-cgmes-conversion` and `powsybl-triple-store-impl-rdf4j`  are for importing a CGMES file,
+- `slf4j-simple` allows you to have simple logging capabilities.
+
+### Layout parameters
+For both example networks, we need to initialize a few layout parameters before generating diagrams (see [SVG layouts]() API guide for more info):
+
+```java
+// Instantiating the default style component library
+ComponentLibrary componentLibrary = new ResourcesComponentLibrary("/ConvergenceLibrary");
+
+// fully automatic layout
+VoltageLevelLayoutFactory voltageLevelLayoutFactory = new PositionVoltageLevelLayoutFactory(new PositionByClustering());
+
+// create default parameters for the SVG layout
+LayoutParameters layoutParameters = new LayoutParameters();
+
+// display line ID instead of line name
+boolean usename = false;
+
+// prefix used for the IDs of all diagram components
+String prefix = "";
+```
+
+## Diagrams from a test network
+We first load the node/breaker test `Network` we are interested in:
+```java
+Network network = FictitiousSwitchFactory.create();
+```
+
+### Generating a voltage level diagram
+We generate a `VoltageLevelDiagram` for voltage level N and then the corresponding SVG diagram file:
+```java
+// create diagram for the voltage level N
+VoltageLevelDiagram voltageLevelDiagram = VoltageLevelDiagram.build(new NetworkGraphBuilder(network), "N", voltageLevelLayoutFactory, usename);
+
+// generate SVG
+voltageLevelDiagram.writeSvg(prefix
+    new DefaultSVGWriter(componentLibrary, layoutParameters),
+    new DefaultDiagramLabelProvider(network, componentLibrary, layoutParameters),
+    new NominalVoltageDiagramStyleProvider(network),
+    Paths.get("/tmp/n.svg"));
+```
+
+We obtain the following SVG:
+
+![N_voltageLevel](img/single-line-diagram_svg-basics/example_n.svg)
```

Comment:
I think you should create a sub-folder:
`img/single-line-diagram/svg-basics/example_n.svg`

Same for the documentation pages. Don't forget to ensure the consistency between the name of the page, and the folder where you put the images